libraries/AP_Scripting/applets/Heli_IM_Col_Tune.lua

```diff
@@ -0,0 +1,140 @@
+-- This script is a useful tool when first setting up a heli.  You can use it to dial in the IM_STB_COL_2 and 
+-- IM_STAB_COL_3 parameters in a more intuative way.  It is meant for use with a transmitter that has two pots
+-- available.  1 Pot can be used to control the gradient of the line between the 40% and 60% curve points.  Use 
+-- this pot to adjust the sensitivity of the collective about the collective midstick.  The 2nd Pot then controls
+-- the value of the 50% point on the curve.  This can be used to set the collective position to aid with hovering 
+-- at the midstick.
+
+-- Tested and working as of 21st Aug 2020 (Copter Dev)
+
+
+function update()
+    local rcin_50 = col50_val_ch:norm_input_ignore_trim()
+    local rcin_grad = col_grad_ch:norm_input_ignore_trim()
+
+    -- Ensure that the channel hasn't been reversed
```

Comment:
Why?

Reply:
Fair, probably best to leave that option to the user.  Removed